documentation/src/main/asciidoc/stories/assembly_server_encryption.adoc

```diff
@@ -0,0 +1,25 @@
+[id='endpoint-encryption']
+:context: server
+= Encrypting {brandname} Server Connections
+
+You can secure {brandname} Server connections by providing cryptographic identities that allow clients to verify {brandname}.
```

Comment:
I would remove "cryptographic identities" and combined the first  two sentences. Something like:

"You can secure {brandname} Server connections using TLS by configuring a keystore that contains the server's public and private keys."